src/code-block-writer.ts

```diff
@@ -1,5 +1,5 @@
-import {stringRepeat, escapeForWithinString} from "./utils/stringUtils";
-import {CommentChar} from "./CommentChar";
+import { stringRepeat, escapeForWithinString, getStringFromStrOrFunc } from "./utils/stringUtils";
```

Comment:
Hmm, this must be my editor :thinking: It would be best to add these to linting rules if we want to enforce either style.

Reply:
No this is fine! Keep it. I'm going to change all these in all my projects to have a space on the start or end. I've been fighting it in my editor recently and I'm going to just go with the space because that seems to be the trend these days.